DEVELOPMENT.md

```diff
@@ -1,63 +1,61 @@
 
-* See [Requirements](https://developers.braintreepayments.com/guides/client-sdk/setup/ios/v4#requirements)
-* iPhone and iPad of all sizes and resolutions and the Simulator
-* CocoaPods
-* `BT` namespace is reserved for Braintree
-* Host app does not integrate the [PayPal iOS SDK](https://github.com/paypal/paypal-ios-sdk)
-* Host app does not integrate with the Kount SDK
-* Host app does not integrate with [card.io](https://www.card.io/)
-* Host app has a secure, authenticated server with a [Braintree server-side integration](https://developers.braintreepayments.com/ios/start/hello-server)
+SPM & Carthage use:
+`#import <BraintreeDropIn/BraintreeDropIn.h>`
+`#import <BraintreeUIKit/BraintreeUIKit.h>`
+`#import <BraintreeCore/BraintreeCore.h>`
 
-## Committing
+CocoaPods uses:
+`#import <BraintreeDropIn/BraintreeDropIn.h>`
+`#import <BraintreeDropIn/BraintreeUIKit.h>`
+`#import <Braintree/BraintreeCore.h>`
 
-* Commits should be small but atomic. Tests should always be passing; the product should always function appropriately.
-* Commit messages should be concise and descriptive.
-* Commit messages may reference the trello board by ID or URL. (Sorry, these are not externally viewable.)
+Public headers for each module must live in the directory `Public/<MODULE_NAME>`. This allows SPM to use the same import syntax as Carthage (e.g., `<BraintreeUIKit/BraintreUIKit.h>`).
 
-## Deployment and Code Organization
+We use if-else preprocessor directives to satisfy each dependency manager. See the below example for importing a **public header file**.
 
-* Code on master is assumed to be in a relatively good state at all times
-  * Tests should be passing, all demo apps should run
-  * Functionality and user experience should be cohesive
-  * Dead code should be kept to a minimum
-* Versioned deployments are tagged with their version numbers
-  * Version numbers conform to [SEMVER](http://semver.org)
-  * These versions are more heavily tested
-  * We will provide support for these versions and commit to maintaining backwards compatibility on our servers
-* Pull requests are welcome
-  * Feel free to create an issue on GitHub before investing development time
-* As needed, the Braintree team may develop features privately
-  * If our internal and public branches get out of sync, we will reconcile this with merges (as opposed to rebasing)
-  * In general, we will try to develop in the open as much as possible
+```objc
+#import <BraintreeDropIn/BTDropInRequest.h> // all package managers, since Pod name same as subspec name
 
-## Releasing
+#ifdef COCOAPODS // CocoaPods
+#import <BraintreeDropIn/BraintreeUIKit.h>
+#else // SPM or Carthage
+#import <BraintreeUIKit/BraintreeUIKit.h>
+#endif
+```
 
-The release process is self-documented in a number of rake tasks.
+To import a Braintree framework written in **Swift** into an Objective-C file, use the following syntax:
+```objc
+#ifdef COCOAPODS // CocoaPods
+#import <BraintreeDropIn/BraintreeDropIn-Swift.h>
```

Comment:
Should this import filename be consistent with the `YourNewSwiftFramework` example naming below?